atomium/templates/system_powered_by/system-powered-by.tpl.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * @file
+ * Contains template file.
+ */
+?>
+<span>
```

Comment:
@drupol I would suggest using **t** function in preprocessing and on template just print the vars, this way all the logic can be treated in preprocess and also easy future changes, avoiding touching the template file as much as possible.
What do you think?
My suggestion would be:
`
function atomium_atomium_theme_system_powered_by(array $existing, $type, $theme, $path) {
  return array(
    'system_powered_by' => array(
      'variables' => array(
        'powered_by_drupal' => t('11Powered by <a href="@poweredby">Drupal</a>', array('@poweredby' => 'https://www.drupal.org')),
        'powered_by_atomium' => t('<a href="@atomium_project_url">Atomium</a>', array('@atomium_project_url' => 'https://www.drupal.org/project/atomium')),
      ),
      'template' => 'system-powered-by',
    ),
  );
}
`

Then in the template:
`
print $powered_by_drupal;
print $powered_by_atomiuml;
`

Reply:
Ok !

Then I would use:

```php
$variables[$variables['base_hook']] = t('Powered by <a href="@poweredby">Drupal</a> & <a href="@atomium_url">Atomium</a>', array('@poweredby' => 'https://www.drupal.org', '@atomium_url' => 'https://drupal.org/project/atomium'))
```

Reply:
Fixed in #346